### DISCUSSION TITLE
Performance questions

### DISCUSSION THREAD

**Discussion by endlessuniverse** [naver/egjs-infinitegrid#473] [Q&A]
Hello there. Thank you for this useful module.
1. Do you recommend the key block in Svelte to recreate the grid when the feed array changes?
2. Do you recommend to feed a large array at once or to paginate it and append the slices sequentially?

Thanks!

**Comment by daybrush** [comment]
@endlessuniverse 

There is no concept of page yet, but there is concept of group.

Even if 1000 items are added, if there is a group, all the items are not rendered.

**Comment by daybrush** [comment]
If an item list with all keys changed comes in, it is likely to be initialized.
